core/src/bundle_stage.rs

```diff
@@ -312,14 +398,23 @@ impl BundleStage {
             ),
         );
 
-        // either qos rate-limited a tx in here or bundle exceeds max cost, drop the bundle
+        // qos rate-limited a tx in here, drop the bundle
         if sanitized_bundle.transactions.len() != num_included {
             QosService::remove_transaction_costs(
                 tx_costs.iter(),
                 transactions_qos_results.iter(),
                 &bank_start.working_bank,
             );
-            qos_service.report_metrics(bank_start.working_bank.clone());
+            warn!(
+                "bundle dropped, qos rate limit. uuid: {} bundle_cost: {},  block_cost: {}",
```

Comment:
extra space before block_cost

Reply:
this is printing current block cost when bundle wouldn't fit

independent of extra space or not